scripts/run_trainer.py

```
@@ -0,0 +1,16 @@
+from dreem.training import train
+from omegaconf import OmegaConf
+import os
```

Ruff

`os` imported but unused

Remove unused import: `os`

(F401)